manifest.php

```diff
@@ -23,10 +23,10 @@
     'label' => 'Delivery schedule',
     'description' => 'An extension is used to visualize the delivery schedule in the form of a calendar.',
     'license' => 'GPL-2.0',
-    'version' => '2.1.1',
+    'version' => '2.2.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => array(
-        'tao' => '>=9.0.0',
+        'tao' => '>=14.3.1',
         'taoDeliveryRdf' => '>=3.6.1'
```

Comment:
Should taoDeliveryRdf version be increased? Looks like constants has been introduced at  v3.12.0